HW 2/css/style.css

```diff
@@ -0,0 +1,130 @@
+body {
+    background-color: #f9f9f9;
+}
+a {
+    text-decoration: none;
+}
+@font-face {
+    font-family: 'OpenSans';
+    src: url(../fonts/OpenSans.ttf);
+}
+.wrapper {
+    margin: auto;
+    width: 1060px;
+    padding: 30px 47px;
+    background-color: #ffffff;
+    box-shadow: 0 0 10px rgba(0,0,0,0.5);
+    font-size: 13px;
+    font-family: 'OpenSans';
+}
+/*HEADER*/
+.header {
+    color: #4f4f4f;
+    font-size: 40px;
+    font-weight: bold;
+}
+.header+div {
+    margin: 30px 0;
+}
+/*TOP MENU*/
+.top-menu {
+    background: linear-gradient(to top, #444343, #5e5e5e);
+    border: 1px solid #e1e1e1;
+    display: flex;
+}
+.top-menu a {
+    padding: 15px;
+    color: #fff;
+    font-size: 15px;
+    border-left: 1px solid rgba(255,255,255, 0);
+    border-right: 1px solid rgba(255,255,255, 0);
+}
+.top-menu a:hover {
+    background: linear-gradient(to top, #2e2e2e, #4a4a4a);
+    border-left: 1px solid rgba(255,255,255, 1);
+    border-right: 1px solid rgba(255,255,255, 1);
+}
+/*LEFT MENU*/
+.left-menu {
+    width: 240px;
+    float: left;
+}
+.left-menu li {
+    border: 1px solid #5e5e5e;
+    border-top: none;
+}
+.left-menu li:first-child {
+    border-top: 1px solid #5e5e5e;
+}
+.left-menu a {
+    color: #656363;
+    display: block;
+    padding: 20px 50px;
+}
+.left-menu li:hover {
+    background: #f9f9f9;
+}
+.left-menu a:hover {
+    position: relative;
+    left: 25px;
+    color: #000;
+}
+/*RIGHT PART, PICTURE+TEXT*/
+.content-right-part {
+    margin-left: auto;
+    width: 678px;
+    margin-bottom: 40px;
+}
+.bike {
+    float: right;
+    border: 1px solid #c8c8c8;
+    padding: 46px 89px 46px 89px;
```

Comment:
why not? u didnt explain. in this case i have float, so img become block element